includes/wikia/tasks/AsyncTaskList.class.php

```diff
@@ -299,19 +299,26 @@ public function queue( AMQPChannel $channel = null ) {
 		] );
 
 		if ( $channel === null ) {
-			$exception = null;
-			$connection = $this->connection();
-			$channel = $connection->channel();
+			$exception = $connection = null;
 			try {
+				$connection = $this->connection();
+				$channel = $connection->channel();
 				$channel->basic_publish( $message, '', $this->getQueue()->name() );
+				$channel->close();
+				$connection->close();
```

Comment:
Overall it seems like a good place to use finally - but not supported in out version of PHP.
You don't need to call close() here because now you do it below anyways.